### PR TITLE
AB#93 NoOrganization Bug

### PR DESCRIPTION
**Pull request by kpjVideo**
Prevents <NoOrganization/> from rendering unless we are sure the user has 0 organizations.

### CODE REVIEW

File: src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -20,7 +23,10 @@ function Dashboard() {
     });
   }, []);
 
-  return <NoOrganization />;
+  if( organizationCount === 0 )
+    return <NoOrganization/>;
+
+  return <></>;
```
Comment (zlix12) [inline src/pages/Dashboard.tsx:29]: Might be a little cleaner to render like so

```suggestion
  return organizationCount === 0 ? <NoOrganization/> : <></>;
```

File: src/pages/Dashboard.tsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import NoOrganization from '../components/NoOrganization';
 import organizationService from '../services/organization.service';
 import { getPreference } from '../services/preference.service';
 
 function Dashboard() {
   const history = useHistory();
+  const [organizationCount, setOrganizationCount] = useState(-1);
```
Comment (zlix12) [inline src/pages/Dashboard.tsx:9]: Could you initialize it like this? The -1 feels a little weird.

```suggestion
  const [organizationCount, setOrganizationCount] = useState<number>();
```

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard.tsx (hunk 1 above)
Comment (kpjVideo) [reply src/pages/Dashboard.tsx]: Dang those ternary operators are clean, why do I always neglect to use them.

**Review by zlix12 — APPROVED** [review]
LGTM

**Pull request merged**